### ISSUE TITLE
What is 'vectorizer_training_dataset'?

### ISSUE THREAD

**Issue by Miopas** [Hi-king/kawaii_creator#1]
I guess it's a dataset of real anime pictures.

**Comment by kogaki** [comment]
Yes. I used about 0.2M illustrations accumulated from web to train the model.

**Comment by Hi-king** [comment]
Yes. I used about 0.2M illustrations accumulated from web to train the model.
